DigitalOffice/Services/TimeManagementService/Validators/CreateWorkTimeRequestValidator.cs

```diff
@@ -0,0 +1,89 @@
+﻿using FluentValidation;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using LT.DigitalOffice.TimeManagementService.Models;
+using LT.DigitalOffice.TimeManagementService.Repositories.Filters;
+using LT.DigitalOffice.TimeManagementService.Repositories.Interfaces;
+using System.Globalization;
+
+namespace LT.DigitalOffice.TimeManagementService.Validators
+{
+    public class CreateWorkTimeRequestValidator : AbstractValidator<CreateWorkTimeRequest>
+    {
+        private IWorkTimeRepository repository;
+
+        /// <summary>
+        /// How many days ago can WorkTime be added.
+        /// </summary>
+        public const int fromDay = 3;
+        /// <summary>
+        /// How many days ahead can WorkTime be added.
+        /// </summary>
+        public const int toDay = 2;
+        /// <summary>
+        /// Limit on working hours in a row.
+        /// </summary>
+        public readonly static TimeSpan workingLimit = new TimeSpan(24, 0, 0); // 24 hours.
```

Comment:
Why not get only property?

Reply:
Unhelpful comment

Reply:
> Why not get only property?

?

Reply:
public fields are bad practice. Will be better: 
`public static TimeSpan workingLimit { get; } = new TimeSpan(24, 0, 0);`

Reply:
Fixed.